lib/css/components/_stacks-lists.less

```diff
@@ -21,8 +21,8 @@
     display: flex;
     flex-wrap: wrap;
 
-        content: '\2022';  // U+2022 BULLET
     > li:not(:last-child):after {
+        content: '•';
```

Comment:
I don't recommend using Unicode in code. This is liable to behave unpredictably depending on whatever default encoding the useragent has. Stick with the `\u` literal